docs/source/amp.rst

```diff
@@ -256,6 +256,106 @@ In this case, combine the two layers using :func:`torch.nn.functional.binary_cro
 or :mod:`torch.nn.BCEWithLogitsLoss`.  ``binary_cross_entropy_with_logits`` and ``BCEWithLogits``
 are safe to autocast.
 
+.. _autocast-xpu-op-reference:
+
+XPU Op-Specific Behavior
+------------------------
+The following lists describe the behavior of eligible ops in autocast-enabled regions.
+These ops always go through autocasting whether they are invoked as part of a :class:`torch.nn.Module`,
+as a function, or as a :class:`torch.Tensor` method. If functions are exposed in multiple namespaces,
+they go through autocasting regardless of the namespace.
+
+Ops not listed below do not go through autocasting.  They run in the type
+defined by their inputs.  However, autocasting may still change the type
+in which unlisted ops run if they're downstream from autocasted ops.
+
+If an op is unlisted, we assume it's numerically stable in ``float16``.
+If you believe an unlisted op is numerically unstable in ``float16``,
+please file an issue.
+
+XPU Ops that can autocast to ``float16``
+""""""""""""""""""""""""""""""""""""""""
+
+``__matmul__``,
```

Comment:
Some of the items in this list are not in the template and supported ops lists. Does that mean they are not supported? `__matmul__` is the example.

Reply:
We have supported mm-related operations.